INSTALL.md

```diff
@@ -57,8 +57,29 @@ Shogun is part of [homebrew-science](https://github.com/Homebrew/homebrew-scienc
     $ sudo brew install shogun
 
 ###Windows <a name="windows"></a>
-Shogun natively compiles under Windows using MSVC, see the [CI build](https://ci.appveyor.com/project/vigsterkr/shogun). We currently do not support a binary
-installer. If you are interested in packaging, documenting, or contributing otherwise, please contact us.
+We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC.We recommend using "Visual Studio 14 2015" which has a good support for C++11. If you do not want to install the whole IDE,you can just use MSBuild to build shogun. Running CMD as administrator with the following commands：
+```
+git clone https://github.com/shogun-toolbox/shogun.git C:\projects\shogun 
+```
+You can change the clone path, we use `C:\projects\shogun` as an example.
+```
+git submodule -q update --init
+```
+updating submodules are required.
+```
+cd C:\projects\shogun
+md build
+cd build
+```
+You need [CMAKE](https://cmake.org/download/) command.
+```
+cmake -G"Visual Studio 14 2015 Win64" -DCMAKE_BUILD_TYPE=Release -DBUILD_META_EXAMPLES=ON -DENABLE_TESTING=ON ..
+```
+This requires [PLY for Python](https://pypi.python.org/pypi/ply) package `python-ply` which you can use `pip install ply` to install it, and [ctags](http://ctags.sourceforge.net/), package `ctags`. Or you can ignore these two packages and use `-DBUILD_META_EXAMPLES=OFF` to continue. You can get the using of `DBUILD_META_EXAMPLES` on the [Examples](#manual-examples)
```

Comment:
`BUILD_META_EXAMPLES` instead of `DBUILD_META_EXAMPLES`.

and for the last sentence use rather like: `For further information regarding the BUILD_META_EXAMPLES parameter please check out the Examples manual...